environments/nonprod/nonp-firewall.auto.tfvars

```diff
@@ -32,13 +32,13 @@ nonprod_firewall = {
         }
       }
       allow-ssh-ingress = {
-        description          = "Allow SSH Connections from the internet"
+        description          = "Allow ingress SSH Connections"
         direction            = "INGRESS"
         action               = "allow"
         ranges               = ["0.0.0.0/0"]
         use_service_accounts = false
         targets              = ["allow-ssh"]
-        sources              = [""]
```

Comment:
this section I am running the example branch - what is running in several of our deployments - will need to sync later with main